src/main/webapp/main/Routes.js

```diff
@@ -2,9 +2,10 @@ import React from "react";
 import { Switch, Route } from "react-router-dom";
 import RolesManagement from "../rolesManagement/RolesManagement";
 import ReviewCategories from "../reviewCategories/ReviewCategories";
-import Profile from "../profile/Profile";
 import Main from "./Main";
 import FundingsSourceReport from "../fundingSourceReport/FundingsSourceReport";
+import DataUseRestrictionDetails from "../dataUse/DataUseRestrictionDetails";
+import Profile from "../profile/Profile";
```

Comment:
nit: We should get in the habit of optimizing imports in JS files which will follow the eslint rules. When you do that, it sorts them appropriately (which actually uses paths, not class names) so we aren't moving lines around for no reason.
```
import React from 'react'
import { Route, Switch } from 'react-router-dom'
import AboutPage from '../aboutPage/AboutPage'
import NewConsentGroup from '../consentGroup/NewConsentGroup'
import DataUseRestrictionDetails from '../dataUse/DataUseRestrictionDetails'
import DataUseRestrictionIndex from '../dataUse/DataUseRestrictionIndex'
import DataUseLetterIndex from '../dataUseLetter/DataUseLetterIndex'
import FundingsSourceReport from '../fundingSourceReport/FundingsSourceReport'
import { LinkWizard } from '../linkWizard/LinkWizard'
import Profile from '../profile/Profile'
import ReviewCategories from '../reviewCategories/ReviewCategories'
import RolesManagement from '../rolesManagement/RolesManagement'
import Main from './Main'

```